django/contrib/postgres/fields/ranges.py

```diff
@@ -41,7 +42,7 @@ def value_to_string(self, obj):
             "lower": value.lower,
```

Comment:
Did you consider using `self.base_field.value_to_string` here instead? It requires some faffery such as is done in `ArrayField` but it is a more general solution to the problem which would work for someone writing their own Range field type as well.